site/components/search/ServiceSearch.tsx

```diff
@@ -1,20 +1,28 @@
 import { Service } from "@create-disruptions-data/shared-ts/disruptionTypes";
+import { Datasource } from "@create-disruptions-data/shared-ts/enums";
 import { Dispatch, ReactElement, SetStateAction, useEffect, useState } from "react";
 import Select, { ControlProps, GroupBase, OptionProps } from "react-select";
 import { getServiceLabel } from "../../utils";
+import Dropdown from "../form/Select";
 
 interface ServiceSearchProps {
     services: Service[];
     setSelectedServices: Dispatch<SetStateAction<Service[]>>;
     selectedServices: Service[];
     reset?: boolean;
+    showToggle?: boolean;
```

Comment:
Do we need this toggle? I think this is the only place this component is used

Reply:
done